netkan/netkan/status.py

```diff
@@ -79,50 +81,64 @@ def export_all_mods(cls, compat: bool = True) -> Dict[str, Any]:
         return data
 
     @classmethod
-    def export_to_s3(cls, bucket: str, key: str, compat: bool = True) -> None:
+    def export_all_mods(cls, compat: bool = True) -> Dict[str, Any]:
+        data: Dict[str, Any] = {}
+        for game_id in ['ksp', 'ksp2']:
+            data.update({game_id: cls.export_game_mods(
+                condition=ModStatus.game_id == game_id, compat=compat)})
+        data.update({'no_game_id': cls.export_game_mods(
+            condition=ModStatus.game_id.does_not_exist(), compat=compat)})
+        return data
+
```

Comment:
On second thought, this isn't called anywhere. Maybe just delete it if we don't need it?